e2e/e2e.js

```diff
@@ -11,33 +11,99 @@ governing permissions and limitations under the License.
 */
 
 const sdk = require('../src/index')
-const util = require("util")
+const path = require('path')
+
+// load .env values in the e2e folder, if any
+require('dotenv').config({ path: path.join(__dirname, '.env') })
 
 var sdkClient = {}
-const tenant = process.env['TARGET_TENANT']
-const apiKey = process.env['TARGET_APIKEY']
-const token = process.env['TARGET_TOKEN']
+const tenant = process.env.TARGET_TENANT
+const apiKey = process.env.TARGET_APIKEY
+const token = process.env.TARGET_TOKEN
 
 test('sdk init test', async () => {
-
   sdkClient = await sdk.init(tenant, apiKey, token)
 
   expect(sdkClient.tenant).toBe(tenant)
   expect(sdkClient.apiKey).toBe(apiKey)
   expect(sdkClient.token).toBe(token)
+})
 
-});
-
-test('test getActivities', async () => {
-  //check success response
-  // var res = await sdkClient.getXTActivityById(321)
+test('test getActivities and getABActivityById', async () => {
   var res = await sdkClient.getActivities()
-  expect(res.total).toBe(9)
+  expect(res.status).toEqual(200)
+  if (res.body.total > 0) {
+    const activities = res.body.activities
+    let processedABActivity = false
+    let processedXTActivity = false
+    // make one call get activity by id for xt and ab type
+    for (let i = 0; i < activities.length; i++) {
+      const element = activities[i]
+      if (element.type === 'ab' && !processedABActivity) {
+        res = await sdkClient.getABActivityById(element.id)
+        expect(res.status).toEqual(200)
+        processedABActivity = true
+      } else if (element.type === 'xt' && !processedXTActivity) {
+        res = await sdkClient.getXTActivityById(element.id)
+        expect(res.status).toEqual(200)
+        processedXTActivity = true
+      }
+    }
+  }
+})
+
+test('test getOffers and getOfferById', async () => {
```

Comment:
`getAudiences` (v3) and `getOffers` (v2) e2e tests: I'm testing the failing case. When I disabled setting the headers in `index.js`, the tests still pass, they should fail, correct? for the other calls that are v1, it is the default so they should pass anyway without setting the headers.

Basically I set headers to undefined here:
```javascript
__createRequest (body, headers) {
    return {
      requestBody: body,
      server: 'https://mc.adobe.io/{tenant-name}/target',
      serverVariables: {
        'tenant-name': this.tenant
      },
      requestInterceptor: req => {
        this.__setHeaders(req, this/*, headers */)
      }
    }
  }
```

Reply:
Had a discussion with Sandeep. Apparently not all of the Adobe Target API functions work the same way. Some will fail if you don't set the correct `Accepts` header for the right api mime-type, and some will. e.g. ` getActivities and getABActivityById` (v3) will fail, but `getAudiences` (v3) and `getOffers` (v2) will not. 

Resolving as a quirk of the API.

Reply:
@sandeep-paliwal @shazron is this documented somewhere on Target API side?

cc @sarahxxu